textpipe/doc.py

```diff
@@ -512,21 +513,48 @@ def aggregate_word_vectors(self, model_name=None, aggregation='mean', normalize=
         else:
             raise NotImplementedError(f'Aggregation method {aggregation} is not implemented.')
 
-    def _load_gensim_word2vec_model(self, model_file=None):
+    def _load_gensim_word2vec_model(self,
```

Comment:
I could be wrong but does this mean that every time we start a component component it loads the model into redis? If that is the case can we add a flag to bypass this behaviour and make the assumption that the model is already present in Redis. 

(Reason that uploading big models to redis will become expensive in datatransfer costs)

Reply:
That's not the case; the loading of the Redis model is a method that has to be called separately. It is not referenced in any other function (including __init__).